crates/rpc/rpc/src/eth/pubsub.rs

```diff
@@ -57,6 +66,7 @@ impl<Provider, Pool, Events, Network> EthPubSub<Provider, Pool, Events, Network>
         chain_events: Events,
         network: Network,
         subscription_task_spawner: Box<dyn TaskSpawner>,
+        api: EthApi<Provider, Pool, Network>,
```

Comment:
we likely don't need this because the pending block stream should be available via the `: Events` type